snappass/main.py

```diff
@@ -22,13 +24,29 @@
 }
 
 
+def check_redis_alive(fn):
+    def inner(*args, **kwargs):
+        try:
+            redis_client.ping()
+        except ConnectionError as e:
+            print(e)
```

Comment:
Nice, how about also printing a friendly error message. I'm debating if we need _only_ a pretty error message like `"Failed to connect to redis"`.
I don't mind using message + stacktrace, but let's make it user friendly.